### ISSUE TITLE
Cryptococcus NTR: respiratory growth phenotype

### ISSUE THREAD

**Issue by Achchuthan** [microbialphenotypes/OMP-ontology#137]
NTR

1)	respiratory growth phenotype
APO cross-refs:
id: APO:0000309
name: respiratory growth
namespace: observable
def: "Growth while utilizing a nonfermentable carbon source." [SGD:mcc]
subset: AspGD
subset: CGD
subset: CryptoGD
subset: SGD
is_a: APO:0000102 ! respiratory metabolism 
2)	altered respiratory growth (is_a respiratory growth phenotype)
3)	increased respiratory growth (is_a altered respiratory growth)
4)	decreased respiratory growth (is_a altered respiratory growth)

**Comment by dianeoinglis** [comment]
I think I'd like to get away from the inferred function. How about 

"growth on nonfermentable carbon source phenotype"

Diane

**Comment by Achchuthan** [comment]
I agree with Diane's suggestion. The term self-explains the phenotype itself.

Thanks,
Achchuthan
